src/treq/test/test_multipart.py

```diff
@@ -80,11 +80,17 @@ def test_unknownLength(self):
         """
         class HasSeek:
             def seek(self, offset, whence):
-                pass
+                """
```

Comment:
This is ok. But at first read I was not sure what we have here.

My suggestion is to rename HasSeek to `HasSeekMissingTell` or just `MissingTell`.

The important part of this class in the context of the test is not that it has seek, but that it's missing a `tell`.

And maybe leave it with a simple a very simple comment and add the docstring to the class.

I know that the docstring was done here to get rid of `pass` :)

Reply:
I renamed them `CantSeek` and `CantTell`. ;)